src/modules/roc_fec/reader.cpp

```diff
@@ -226,7 +236,7 @@ bool Reader::check_packet_(const packet::PacketPtr& pp, size_t pos) {
     roc_panic_if_not(has_source_);
 
     if (!pp->rtp()) {
-        roc_log(LogDebug, "fec reader: repaired unexcpeted non-rtp packet");
+        roc_log(LogDebug, "fec reader: repaired unexpected non-rtp packet");
         return false;
     }
 
```

Comment:
> I still need to use seqnum in the fec reader to ensure that packets via bad
  seqnum wan't be recovered.

```
    if (pp->rtp()->seqnum != packet::seqnum_t(cur_block_sn_ + pos)) {
        roc_log(LogDebug,
                "fec reader: repaired packet has bad seqnum: got=%lu expected=%lu",
                (unsigned long)pp->rtp()->seqnum,
                (unsigned long)packet::seqnum_t(cur_block_sn_ + pos));
        return false;
    }
```

This check is redundant. We can just remove it to avoid a dependency on how the sender fills seqnums.

Reply:
Done